SendGrid integration to send message with subject and body.

subhransu opened the pull request:
Send a message to a person with the subject and body. 

### API POST Request format

<img width="657" alt="Screenshot 2020-05-16 at 9 10 34 PM" src="https://user-images.githubusercontent.com/1495621/82120563-b040c480-97b9-11ea-8752-5689e39a9beb.png">

### Email template (without much formatting) 

<img width="644" alt="Screenshot 2020-05-16 at 9 12 46 PM" src="https://user-images.githubusercontent.com/1495621/82120617-001f8b80-97ba-11ea-9b91-d6db70d29dc5.png">

kajalsinha commented on pkg/web/web.go:
```diff
@@ -31,6 +32,11 @@ func RespondOK(ginCtx *gin.Context, data interface{}) {
 
+func RespondOKWithoutData(ginCtx *gin.Context) {
+	ginCtx.Header("Content-Type", "application/json; charset=utf-8")
+	ginCtx.Status(http.StatusOK)
```
Can make it [204 No Content](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)

subhransu commented:
Modified paremeter names in the last [commit](https://github.com/jobsgowhere/jobsgowhere/pull/76/commits/184b213439380f15eec96a7a5959b37be93ece1f)

* Earlier I was using title (for message title) and subject (for message body)
* Replaced this with subject (for email subject) and body (for message body)

<img width="780" alt="Screenshot 2020-05-17 at 11 52 27 AM" src="https://user-images.githubusercontent.com/1495621/82135399-f6873980-9834-11ea-9823-6f8677434d10.png">
